etc/kayobe/environments/ci-aio/automated-setup.sh

```diff
@@ -88,22 +88,43 @@ kayobe overcloud host configure
 
 kayobe overcloud service deploy
 
+export KAYOBE_CONFIG_SOURCE_PATH=$BASE_PATH/src/kayobe-config
+export KAYOBE_VENV_PATH=$BASE_PATH/venvs/kayobe
+pushd $BASE_PATH/src/kayobe
+./dev/overcloud-test-vm.sh
+
 if $AIO_RUN_TEMPEST; then
+    if type apt; then
+        sudo apt install docker-buildx-plugin
```

Comment:
Don't you need `apt -y`?

Reply:
You appear to have an old version of this change. This issue is already fixed

Reply:
Sorry, I just realised I only looked at the first commit.